tests/unit_tests/test_server_client.py

```diff
@@ -737,6 +737,90 @@ def add_one(x: int) -> int:
         }
 
 
+@pytest.mark.asyncio
+async def test_astream_log_allowlist(event_loop: AbstractEventLoop) -> None:
+    """Test async stream with an allowlist."""
+
+    async def add_one(x: int) -> int:
```

Comment:
Would it make sense to run this test with a slightly longer chain composed of 2-3 steps with names?